openslides_backend/action/actions/meeting/update.py

```diff
@@ -295,3 +307,12 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
             )
             if not is_superadmin:
                 raise MissingPermission(OrganizationManagementLevel.SUPERADMIN)
+
+    def get_committee_id(self, meeting_id: int) -> int:
+        if not hasattr(self, "__committee_id"):
```

Comment:
Why this `__committee_id` cache var? I think it is not needed, as the ds should handle this.

Reply:
> Why this `__committee_id` cache var? I think it is not needed, as the ds should handle this.

The datastore adapter don't cache it, because lock_result = False, see https://github.com/OpenSlides/openslides-backend/blob/499cef34dad17578ee0c5141bb164df2be40e709/openslides_backend/services/datastore/cache_adapter.py#L62
The datastore itself is an external service